### DISCUSSION TITLE
problems about pip install commonroad-scenario-designer on windows

### DISCUSSION THREAD

**Discussion by myj8888** [CommonRoad/commonroad-scenario-designer#24] [Q&A] [answered]
I had some problems executing pip install commonroad-scenario-designer  with annaconda, as shown in the picture

<img width="840" alt="problem" src="https://github.com/user-attachments/assets/1d74f6a0-ad07-4fab-8deb-77a4d6de346b">

**Comment by smaierhofer** [comment]
the scenario-designer is currently not-supported for Windows. 
Please use a linux-based OS or the windows subsystem for linux.

**Reply by myj8888** [reply]
Thank you for your answer！ I tried linux，but some problems occurred  as shown in the picture. Please help me again,thanks.
<img width="637" alt="问题1" src="https://github.com/user-attachments/assets/8ccc8265-8b8b-4528-9fa4-41c3af6b327a">

**Comment by smaierhofer** [comment, marked as answer]
maybe try to install gcc

which python version are you using?

**Reply by myj8888** [reply]
python 3.8

**Comment by myj8888** [comment]
when I install gcc,the problem has solved.thanks
